Cython/Compiler/Pipeline.py

```diff
@@ -222,6 +222,7 @@ def create_pipeline(context, mode, exclude_classes=()):
         DropRefcountingTransform(),
         FinalOptimizePhase(context),
         GilCheck(),
+        FudgeCppTemps(context),
```

Comment:
I think running the plain checks as the very last step is better.
```suggestion
        CoerceCppTemps(context),
        GilCheck(),
```